index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hello, Rachel</title>
+  <link rel="stylesheet" href="main.css">
+</head>
+    <title> Embarking under the markings! </title>
```

Comment:
Make sure to use <https://validator.w3.org/>! It would've caught the fact that you closed the `</head>` tag twice (and then tell you that you used the `<title>` tag twice.